### PR TITLE
Support multiple templates folders

### PR DESCRIPTION
**Pull request by drscream**
Provide support to append additional templates folders for IONIT. With this change it's not required to run multiple ionit commands per templates folder because they could be appended in the command line.

Example:

```
  ionit -c tests/config/static \
        -t tests/template/static \
        -t tests/template/static2/
```

This feature is requested to run one ionit service for `/etc` and `/opt` templates at the same time.

### CODE REVIEW

File: ionit [hunk 1]
```diff
@@ -207,8 +208,8 @@ def main(argv):
     parser.add_argument(
         "-t",
         "--templates",
-        default="/etc",
-        help="Directory to search for Jinja templates (default: %(default)s)",
+        action="append",
+        help="Directory to search for Jinja templates " f"(default: {DEFAULT_TEMPLATES})",
```
Comment (bdrung) [inline ionit:212]:
```suggestion
        help=f"Directory to search for Jinja templates (default: {DEFAULT_TEMPLATES})",
```

File: ionit [hunk 2]
```diff
@@ -34,6 +34,7 @@ except ImportError:
 
 
 DEFAULT_CONFIG = "/etc/ionit"
+DEFAULT_TEMPLATES = "/etc"
```
Comment (bdrung) [inline ionit:37]: Since the name is using the plural form, it should be a list.

File: ionit [hunk 3]
```diff
@@ -209,7 +211,7 @@ def main(argv):
         "-t",
         "--templates",
         action="append",
-        help="Directory to search for Jinja templates " f"(default: {DEFAULT_TEMPLATES})",
+        help=f"Directory to search for Jinja templates (default: {str(DEFAULT_TEMPLATES)[2:-2]})",
```
Comment (bdrung) [inline ionit:214]: That string conversion looks ugly. So better dropping the idea of `DEFAULT_TEMPLATES` being a list and rename `DEFAULT_TEMPLATES` to `DEFAULT_TEMPLATE` to just contain `"etc"`.

Comment (bdrung) [reply ionit:214]: Or maybe for consistency: `DEFAULT_TEMPLATES_DIRECTORY`.

**Comment by bdrung** [comment]
There is no test case for using the default template directory (which causes a decline in coverage).

**Comment by drscream** [comment]
Do you have any recommendation how to implement the test for the default directory?

**Comment by drscream** [comment]
> Do you have any recommendation how to implement the test for the default directory?

I hope it will match the current requirements now.

**Comment by bdrung** [comment]
Thanks for the contribution. I will cut a new release.

**Pull request merged**